prime-router/src/main/kotlin/cli/LivdTableDownload.kt

```diff
@@ -157,24 +179,13 @@ class LivdTableDownload : CliktCommand(
 
         val data = StringBuffer() // Buffer and output file for CSV data
         val fileInputStream = FileInputStream(File(inputfile))
-        var workbook: Workbook? = null
         val ext: String = FilenameUtils.getExtension(inputfile)
-        if (ext.equals("xlsx", ignoreCase = true)) {
-            workbook = XSSFWorkbook(fileInputStream)
-        } else if (ext.equals("xls", ignoreCase = true)) {
-            workbook = HSSFWorkbook(fileInputStream)
-        }
-
-        // Check for output file duplication
-        if (File(outputfile).exists()) {
-            val c = TermUi.prompt("\t$outputfile file is already existed: You want to overwrite it (y/n)?", "n")
-            if (c?.lowercase() == "n") {
-                return false
-            } else {
-                TermUi.echo("\tOverwriting the $outputfile file.")
-            }
+        if (!ext.equals("xlsx", ignoreCase = true)) {
```

Comment:
The code in extractLivdTable seems very complicated - how do we know it works?   How is it tested?    It needs either a unit test or an integration/smoke test (a CoolTest()).

Seems like you could write a CoolTest() where you do an actual-vs-expected data comparison.   That is, a test that reads in an existing LIVD, then exports the data from the table, and compares that against a known good export that **has been checked by a human and confirmed good.**

The LIVD data is pretty central to what we do with Covid-19.